src/improvements/tasks/sep/017-1-U16-opcm-upgrade-v400-base-approveHash/config.toml

```diff
@@ -0,0 +1,5 @@
+l2chains = [{ name = "Base Sepolia", chainId = 84532 }]
+templateName = "GnosisSafeApproveHash"
+
```

Comment:
I think it's probably best to include nonces in this file and also the other config toml file to ensure the hashes don't change.